lib/matplotlib/ticker.py

```diff
@@ -1002,6 +1008,12 @@ def __init__(self, base, offset):
         self._base = base
         self.offset = offset
 
+    def set_params(self, **kwargs):
+        if 'base' in kwargs:
+            self.base = kwargs['base']
```

Comment:
I think this has to be `_base` with an underscore; or remove the underscore elsewhere in this class.